ui/app/AppLayouts/Onboarding/views/InsertDetailsView.qml

```diff
@@ -158,6 +158,7 @@ Item {
 
         StyledText {
             id: chatKeyTxt
+            objectName: "InsertDetailsView_chatKeyTxt"
```

Comment:
nit: for others we use camelCase: insertDetailsViewChatKeyTxt

Reply:
I was trying to add context by separating components/context to be easy to track and didn't consider it to follow any coding conventions. Thanks for the heads up. Fixed it.